src/server/getters/get-dispute-info.ts

```diff
@@ -0,0 +1,76 @@
+import { parallel } from "async";
+import * as Knex from "knex";
+import * as _ from "lodash";
+import { Address, MarketsRowWithCreationTime, OutcomesRow, AsyncCallback, Payout, UIStakeInfo } from "../../types";
+import { getMarketsWithReportingState, normalizePayouts } from "./database";
+
+interface DisputesResult {
+  markets: Array<MarketsRowWithCreationTime>;
+  disputes: Array<OutcomesRow>;
+  crowdsourcers: Array<StakeRow>;
+  initialReport: Array<StakeRow>;
+}
+
+interface StakeRow extends Payout {
+  marketID: Address;
+  amountStaked: number;
+  payoutID: number;
+  size?: number;
+  completed: number|null;
+}
```

Comment:
can number handle the size of attoREP? could be 10x-18 long, right.

Reply:
you are correct, it is not the correct numeric container. we have a different CH for going through fixing all these, but i'll go ahead and do this one right now

Reply:
Actually, it looks like this might be a bit large to do correctly in the same PR, it should be a different PR since we need to change the getter mechanisms from the DB (and make it work with both postgres and sqlite), or else the number could get squashed on its way out from DB.

I could just half-ass it right now and bignumber/toFixed() it on the way out so it doesn't change for you when we fix it for realz?

Reply:
is that storing as hex like straight from contract log? either way you da man.

Reply:
no we will serve numbers as base-10, strings